### ISSUE TITLE
password file cannot be located outside of `host_system_mounting_directory`

### ISSUE THREAD

**Issue by marrip** [InPreD/TSOPPI_SADET#14]
It would be nice to be able to place it anywhere and not in the same path as it is not part of the "results" of TSOPPI. ☺️

**Comment by danielvo** [comment]
Well, "anywhere". It has to be placed in a volume that is mounted/visible inside the image (?). The tool does assume that a single volume is mounted (and all the input and output should be located somewhere within it), which is perhaps the issue? We can discuss volume mounting/handling in general.

**Comment by marrip** [comment]
true, I added it manually to the `/home` directory and was surprised when I got the error. I have now moved it to the "correct" place. I think the handling of paths is a bit too rigid and would benefit from allowing for more configuration options. 🙂

**Comment by danielvo** [comment]
In the extreme case, one might wish to have each input file/directory placed on a different volume (and the output could be directed to a yet different volume), and then all those volumes would need to be made visible/accessible inside the image. There might be a simpler and/or more flexible way to handle the path juggling and conversions, so I am open for suggestions.
